data/logs_model/splunk_field_mapper.py

```
@@ -0,0 +1,315 @@
+        for username in usernames:
+            if username in self._user_cache:
+                username_user_map[username] = self._user_cache[username]
+            else:
+                usernames_to_lookup.append(username)
+
+        for username in usernames_to_lookup:
+            try:
+                user = model.user.get_namespace_user(username)
+                self._user_cache[username] = user
+                username_user_map[username] = user
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)